src/mixins/onPageLoadScrollToFragment.js

```diff
@@ -8,14 +8,12 @@
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-import { waitFrames } from 'docc-render/utils/loading';
 import scrollToElement from 'docc-render/mixins/scrollToElement';
 
 export default {
   mixins: [scrollToElement],
-  async mounted() {
```

Comment:
Should this still be `async` and just await the `scrollToElement` call instead since that is now asynchronous?

Reply:
We dont need to await it, unless we are doing something explicitly after it.